test/opt-test.js

```diff
@@ -81,7 +81,14 @@ describe("TidyOption:", function() {
       expect(doc.optGet("add-xml-decl")).to.be.false;
       expect(function() {
         doc.optSet("add-xml-decl", "some strange value");
-      }).to.throw;
+      }).to.throw();
+    });
+
+    it("setting a readonly option throw", function() {
+      var doc = TidyDoc();
+      var opt = doc.getOption(8);
```

Comment:
Better refer to the option by name, for reasons [explained in the API doc](https://github.com/gagern/node-libtidy/blob/master/API.md#tidyoptionid).